models/login_source.go

```diff
@@ -416,6 +416,19 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 		return nil, ErrUserNotExist{0, login, 0}
 	}
 
+	// Update User if exist
+	isExist, err := IsUserExist(0, sr.Username)
+	if err != nil {
+		return nil, err
+	} else if isExist {
+		user.FullName = composeFullName(sr.Name, sr.Surname, sr.Username)
```

Comment:
It seems it doesn't update the information to database?

Reply:
See line 418